.travis.yml

```
@@ -43,7 +45,7 @@ deploy:
     - release/goss-linux-arm.sha256
     - extras/dgoss/dgoss
     - extras/dgoss/dgoss.sha256
-  skip_cleanup: true
+  cleanup: false
```

```suggestion
  skip_cleanup: true
```
Seems you and I went back and forth on this. Last time this was changed, it prevented me from cutting a release :(

As far as I can tell the documentation still says to use `skip_cleanup`
https://docs.travis-ci.com/user/deployment

`cleanup` seems to be part of the beta deployer, which goss isn't currently using:
https://docs.travis-ci.com/user/deployment-v2

Once deployment-v2 is no longer in beta, we can make this change.